pages/checkout.vue

```diff
@@ -275,16 +228,16 @@ export default {
     // TODO
     // const applyPromocode = async (couponApplied) => await applyCoupon(couponApplied)
 
-    const isValidCoupon = computed(() => !getOrder.value?.invalidCoupons[0]?.couponCode)
+    // const isValidCoupon = computed(() => !getOrder.value?.invalidCoupons[0]?.couponCode)
 
-    const invalidCouponErrorText = computed(
-      () =>
-        `${getOrder.value?.invalidCoupons[0]?.couponCode} ${context.root.$t("is an invalid code")}`
-    )
+    // const invalidCouponErrorText = computed(
```

Comment:
Delete the code if uneeded, don't comment out and leave

Reply:
Resolved.